alan-hadyk/grocify
# Initial mobile client setup

# Remarks

1. During code review, please ignore the following auto-generated files and folders:
- `client-mobile/ios` (build with iOS version)
- `client-mobile/src/api/schema.ts` (API Schema)

2. Certain files and components are dummy code (usually marked with comment similar to `TODO - Dummy`).

# Testing scenario 1

1. Follow `Prerequisites` and `Development - iOS simulator` instructions in `client-mobile/README.md`
2. Make sure that the app runs on the iOS simulator. Try to move around, switch between tabs and use the API (via black buttons).

![Zrzut ekranu 2023-10-8 o 19 28 15](https://github.com/alan-hadyk/grocify/assets/4722055/d1335924-7b35-4b2a-99d7-16f2ebbeb864)

3. Try to run E2E tests by following `E2E tests` section in `client-mobile/README.md`

# Testing scenario 2

1. Follow `Prerequisites` and `Development - Expo app (iOS)` instructions in `client-mobile/README.md`
2. Make sure that the app runs on the actual mobile device via `Expo Go` app. Try to move around, switch between tabs and use the API (via black buttons).
